core/src/serve_repair.rs

```diff
@@ -420,6 +434,20 @@ impl ServeRepair {
                         Self::report_reset_stats(&me, &mut stats);
                         last_print = Instant::now();
                     }
+                    const INTERVAL_MS: u64 = 1000;
+                    const MAX_BYTES_PER_SECOND: usize = 10_000 * 1200;
+                    const MAX_BYTES_PER_INTERVAL: usize =
+                        MAX_BYTES_PER_SECOND * INTERVAL_MS as usize / 1000;
+                    const MAX_BYTES_BUDGET: usize = MAX_BYTES_PER_INTERVAL * 5;
```

Comment:
For better code organization and more readability can you plz move this constants as first thing inside the function.

Reply:
yup, for sure